test/fixtures/userStatus/userStatus.js

```diff
@@ -12,6 +12,19 @@ const userStsDataForNewUser = {
   },
 };
 
+const oooUserStsDataForShortDuration = {
+  currentStatus: {
+    message: "",
+    state: "OOO",
+    updatedAt: 1673893800000,
+    from: 1673893800000,
+    until: 1674066600000,
+  },
+  monthlyHours: {
+    updatedAt: 1668215609000,
+    committed: 40,
```

Comment:
Why are we passing monthlyHours when setting OOO?
Am I missing something here

Reply:
this data I have used it here for testing if the endpoint is able to store the data in the collection document when requested by an authorized user or superuser. The test will work with both the partial status data as well as status data with monthly hours. Few of the tests are using complete data while few are using partial data.